### DISCUSSION TITLE
No XCH in my wallet after reinstalling Window 10

### DISCUSSION THREAD

**Discussion by Mareknown** [Chia-Network/chia-blockchain#2180] [Q&A]
Yesterday I have changed HDD to SSD and install a new Windows 10 on it. I installed the Chia 1.05 version (GUI) and log in to my account using the 24 words. To my surprise the wallet is showing 0 XCH while I had 2 in my wallet. Will they appear when the synchronization is finished? The old plots are not synchronized yet either. I still have the HDD disc with the “old” Windows and Chia installed. Shall I swap them, log in to the previous Chia and transfer the XCH…but how? Please help me, it is my wife machine, she is going to “kil!” me if I do not retrieve her XCH.

**Comment by hoffmang9** [comment]
Your rewards are on the chain and not on your computer. They will appear when the wallet gets in sync.

**Reply by Mareknown** [reply]
Uff, thanks God! Many thanks for your prompt response.
